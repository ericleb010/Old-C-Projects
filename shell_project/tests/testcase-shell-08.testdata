echo man
echo Hello
echo Fancy meeting you here!
