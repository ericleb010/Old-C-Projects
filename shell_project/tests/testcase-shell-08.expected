man
Child (<0>) exited -- success (Status: 0)
Hello
Child (<0>) exited -- success (Status: 0)
Fancy meeting you here!
Child (<0>) exited -- success (Status: 0)

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 
